distributed/worker.py

```diff
@@ -609,7 +545,7 @@ def __init__(
         profile_cycle_interval = parse_timedelta(profile_cycle_interval, default="ms")
         assert profile_cycle_interval
 
-        self._setup_logging(logger)
+        self._setup_logging(logger, wsm_logger)
```

Comment:
Should I also add `distributed.worker_memory.logger`?

Reply:
I suggest a follow up PR. I have thoughts on logging and would prefer this to be in a dedicated PR